webapp/src/pxtjson.tsx

```diff
@@ -82,7 +91,9 @@ export class Editor extends srceditor.Editor {
                             onChange={() => applyUserConfig(uc) } />
                     ) }
                     <sui.Field>
+                        <sui.Button text={lf("Save")} class={`green ${this.isSaving ? 'disabled' : ''}`} onClick={() => save()} />
                         <sui.Button text={lf("Edit Settings As text") } onClick={() => this.parent.editText() } />
+                        <sui.Button text={lf("Delete")} class={`red right floated`} onClick={() => deleteProject()} />
```

Comment:
Really hard to discover where the "delete project" button is when only in settings.